src/main/java/uk/gov/hmcts/reform/iacaseaccessapi/infrastructure/service/CcdSupplementaryDetailsSearchService.java

```diff
@@ -1,34 +1,85 @@
 package uk.gov.hmcts.reform.iacaseaccessapi.infrastructure.service;
 
-import java.util.ArrayList;
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.iacaseaccessapi.domain.entities.SupplementaryDetails;
 import uk.gov.hmcts.reform.iacaseaccessapi.domain.entities.SupplementaryInfo;
 import uk.gov.hmcts.reform.iacaseaccessapi.domain.services.SupplementaryDetailsService;
+import uk.gov.hmcts.reform.iacaseaccessapi.infrastructure.clients.CoreCaseDataApi;
+import uk.gov.hmcts.reform.iacaseaccessapi.infrastructure.clients.Query;
+import uk.gov.hmcts.reform.iacaseaccessapi.infrastructure.clients.model.ccd.CaseDetails;
+import uk.gov.hmcts.reform.iacaseaccessapi.infrastructure.clients.model.ccd.SearchResult;
+import uk.gov.hmcts.reform.iacaseaccessapi.infrastructure.security.SystemTokenGenerator;
+import uk.gov.hmcts.reform.iacaseaccessapi.infrastructure.security.oauth2.IdentityManagerResponseException;
 
 @Service
 @Slf4j
 public class CcdSupplementaryDetailsSearchService implements SupplementaryDetailsService {
+    private static final String CASE_TYPE_ID = "Asylum";
+    private static final String APPELLANT_FAMILY_NAME = "appellantFamilyName";
+    Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
because you have @Slf4j annotation you do not need to add second `logger` field, you can use `log` field having only annotation

Reply:
logger field removed